src/nexusrpc/_util.py

```diff
@@ -0,0 +1,133 @@
+# See
+# https://docs.python.org/3/howto/annotations.html#accessing-the-annotations-dict-of-an-object-in-python-3-9-and-older
```

Comment:
3.9 will be EOL by October. I'd be ok if we went ahead and said Python Nexus is 3.10+ and remove any 3.9 polyfills/workarounds